Mathlib/Analysis/NormedSpace/AffineIsometry.lean

```diff
@@ -847,10 +847,7 @@ theorem AffineMap.continuous_linear_iff {f : P →ᵃ[𝕜] P₂} : Continuous f
       (AffineIsometryEquiv.vaddConst 𝕜 <| f default).toHomeomorph.symm ∘
         f ∘ (AffineIsometryEquiv.vaddConst 𝕜 default).toHomeomorph := by
     ext v
-    -- Porting note: was just `simp`
-    simp only [(AffineIsometryEquiv.coe_toHomeomorph), Function.comp_apply, (AffineMap.map_vadd),
-      eq_self_iff_true, (AffineIsometryEquiv.toHomeomorph_symm),
-      (AffineIsometryEquiv.coe_vaddConst), (vadd_vsub), (AffineIsometryEquiv.coe_vaddConst_symm)]
+    simp
```

Comment:
I wonder if this one will make a performance difference. Benchmarking will tell.